tests/unit_tests/test_all_icons.py

```diff
@@ -110,7 +111,9 @@ def test_all_icon_sets_endpoint(self):
         """
         Checking that the endpoint /sets returns all available icon sets
         """
-        response = self.app.get(f"{ROUTE_PREFIX}/sets", headers=self.default_header)
+        response = self.app.get(
+            url_for('all_icon_sets', _external=True), headers=self.default_header
```

Comment:
Don't use _external=True in unittest to call the route, this is not needed as the unittest flask client is not a real client it is just a mocking. The `_external=True` in unittest should only be used to test the `url` fields in answer.